package.json

```diff
@@ -715,7 +715,7 @@
         "@azure/arm-subscriptions": "^5.1.0",
         "@azure/container-registry": "^1.1.0",
         "@azure/core-auth": "^1.8.0",
-        "@azure/identity": "^4.8.0",
+        "@azure/identity": "4.6",
```

Comment:
Minor do we want to keep the whole 4.6.0 but just a bit and if it works I am merging it 😊☕️